app/templates/public/exhibition/view.hbs

```diff
@@ -5,7 +5,7 @@
         </div>
         <br/>
         <h1 class="ui centered header no margin">
-            <img class="ui small image m-0 mb-1" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo">
+            <img class="ui image m-0" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo" style="height:250px; width:250px; padding-bottom:60px;">
```

Comment:
```suggestion
              <img style="height: 30px; width: auto; object-fit: contain;" class="ui small image m-0 mb-1" src="{{or this.model.exhibitor.logoUrl '/images/placeholders/Other.jpg'}}" alt="Logo">
```

Reply:
![Screenshot from 2021-03-08 03-10-31](https://user-images.githubusercontent.com/61330148/110255905-236f2d80-7fbc-11eb-9837-7c2d4b37966c.png)

Reply:
Changes are done mam, Thank you so much for your help.


![Screenshot (1087)](https://user-images.githubusercontent.com/63748249/110276239-b9c14480-7ff8-11eb-9f9e-50cf90735ef9.png)

Reply:
@MeghalBisht Mam Sorry to disturb you, Could you please review this PR.